homeassistant/const.py

```diff
@@ -237,6 +237,7 @@
 
 # Electrical attributes
 ATTR_VOLTAGE = 'voltage'
+ATTR_CURRENT = 'current'
```

Comment:
Let's not add this to the main const file but instead keep it inside the deconz platform.

Reply:
Sure